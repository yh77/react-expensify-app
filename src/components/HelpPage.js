import React from 'react';

const HelpPage = () => (
    <div>
        My help component.
    </div>
);

export default HelpPage;

