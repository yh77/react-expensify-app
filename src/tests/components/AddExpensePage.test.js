import React from 'react';
import {shallow} from 'enzyme';
import { AddExpensePage } from '../../components/AddExpensePage';

test ('test rendering AddExpensePage', () => {
    const addExpense = jest.fn();
    const history = {push: jest.fn()};
    const wrapper = shallow(<AddExpensePage addExpense={addExpense} history={history}/>);
    expect(wrapper).toMatchSnapshot();

});
