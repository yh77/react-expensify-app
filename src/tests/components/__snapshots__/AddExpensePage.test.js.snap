// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test rendering AddExpensePage 1`] = `
<div>
  <h1>
     Add Expense 
  </h1>
  <ExpenseForm
    onSubmit={[Function]}
  />
</div>
`;
